### PR TITLE
Add files via upload

### LINKED ISSUE #5: Configurable Timer interval slider
I'm thinking that we could implement a feature where a user is able to configure the interval (15s - 90s) in between songs with a [slider](https://material-ui.com/components/slider/).

And in terms of where we could put the slider, I'm thinking that it could go on the right side of the screen (see image) and we could push the device dropdown and playlist search bar to take up the left half of the screen. Let me know what your thoughts are @dwilliams27 

image

### PR DESCRIPTION
**Pull request by ColemanMitch**
Added the configurable interval slider to the `partyStarted` screen with the Timer. Take a look when you have a second. Sometimes the timer skips and speeds up... not sure _exactly_ how to recreate it

### CODE REVIEW

**Comment by dwilliams27** [comment]
Imma check this out now

File: build/static/js/3.d91b8271.chunk.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
...essingStart-t.startTime,f.entries.push(t),p.add(f),n())},l=c("first-input",v);n=m(t,f,e),l&&s((function(){l.takeRecords().map(v),l.disconnect()}),!0),l&&d((function(){var r;f=u("FID"),n=m(t,f,e),o=[],a=-1,i=null,F(addEventListener),r=v,o.push(r),T()}))},C=function(t,e){var n,i=y(),a=u("LCP"),r=function(t){var e=t.startTime;e<i.timeStamp&&(a.value=e,a.entries.push(t)),n()},o=c("largest-contentful-paint",r);if(o){n=m(t,a,e);var f=function(){p.has(a)||(o.takeRecords().map(r),o.disconnect(),p.add(a),n())};["keydown","click"].forEach((function(t){addEventListener(t,f,{once:!0,capture:!0})})),s(f,!0),d((function(i){a=u("LCP"),n=m(t,a,e),requestAnimationFrame((function(){requestAnimationFrame((function(){a.value=performance.now()-i.timeStamp,p.add(a),n()}))}))}))}},P=function(t){var e,n=u("TTFB");e=function(){try{var e=performance.getEntriesByType("navigation")[0]||function(){var t=performance.timing,e={entryType:"navigation",startTime:0};for(var n in t)"navigationStart"!==n&&"toJSON"!==n&&(e[n]=Math.max(t[n]-t.navigationStart,0));return e}();n.value=n.delta=e.responseStart,n.entries=[e],t(n)}catch(t){}},"complete"===document.readyState?setTimeout(e,0):addEventListener("pageshow",e)}}}]);
```
Comment (dwilliams27) [inline build/static/js/3.d91b8271.chunk.js:1]: All files in 'build/' should be in the .gitignore, no need to commit these

**Comment by dwilliams27** [comment]
yoo this feature works well man! nice work

**Comment by dwilliams27** [comment]
Remove those builds files and this looks good to merge in

**Comment by ColemanMitch** [comment]
> Remove those builds files and this looks good to merge in

Is there an easy way to remove all of the files in the `/build` folder from GitHub other than pushing another commit to this PR with the `/build` folder in the `.gitignore`? In other words, can I manually select and delete all the `/build` files from the desktop client of GitHub? @dwilliams27

**Comment by dwilliams27** [comment]
Not sure, I almost never use to UI for making any file changes. Definitely should one way or another get that folder into the .gitignore and the files deleted

**Comment by ColemanMitch** [comment]
Close with latest commit to main

**Pull request closed without merging**